### PR TITLE
Use a signed byte for the dummy value in AvailableRandomizationUnits.

### PR DESCRIPTION
**Pull request by rfk**
Unsigned types are currently experimental in Kotlin, and can only be
used by consumers if they explicitly opt in. Let's not put this opt-in
requirement on all consumers of this SDK if we can avoid it.

### CODE REVIEW

**Comment by rfk** [comment]
@dmose requesting r? from you in case this change would affect the desktop work in unexpected ways.

**Review by dmose — APPROVED** [review]
I'm not aware of any issues that would cause (famous last words), especially since it's a dummy.  I'd be interested to hear if @eoger knows something that I don't, but I'll let you decide whether that should block this from landing.  r=dmose

**Comment by mhammond** [comment]
FWIW, I chose `u8` at random here when adding it, so I don't see why `i8` isn't fine.

**Comment by rfk** [comment]
> FWIW, I chose u8 at random here when adding it, so I don't see why i8 isn't fine.

In that case, let's ship it :-)

**Pull request merged**
